src/ws-dropdown/dropdown-menu-item.js

```diff
@@ -89,9 +93,16 @@ export class DropdownMenuItem extends Component {
       this.props.handle('show-child', this.menu);
     } else {
       if (!this.context.multiple) {
-        this.state.selected = true;
-        this.props.handle('change', this.state);
+        // If it is selected we publish null because it will be deselected in the upper menu
+        if (this.state.selected) {
+          this.props.handle('change', null);
```

Comment:
Nope. I'm pretty sure that we don't need to do it. The flag stored is an indicator that the item is selected to be presented as a value or stored in values list.